support acs gen2 (#6402)
Azure/azure-cli-extensions: src/spring/HISTORY.md

Release History
===============
1.14.0
---

> * Add new command -- `az spring application-configuration-service create --generation` to support create Application Configuration Service with different generation
> * Add new command -- `az spring application-configuration-service update --generation` to support update Application Configuration Service to different generation
> * Add new command -- `az spring application-configuration-service git repo add --ca-cert-name` to support bind certificate to Application Configuration Service Gen2

Reviewer: ```suggestion
* Add new command -- `az spring application-configuration-service create --generation` to support creating Application Configuration Service with different generation
* Add new command -- `az spring application-configuration-service update --generation` to support updating Application Configuration Service to different generation
* Add new command -- `az spring application-configuration-service git repo add --ca-cert-name` to support binding certificate to Application Configuration Service Gen2
```

Author: updated

> * Add new command -- `az spring application-configuration-service create --generation` to support creating Application Configuration Service with different generation
> * Add new command -- `az spring application-configuration-service update --generation` to support updating Application Configuration Service to different generation
> * Add new command -- `az spring application-configuration-service git repo add --ca-cert-name` to support binding certificate to Application Configuration Service Gen2

Reviewer 2: ```suggestion
* Add new command `az spring application-configuration-service create --generation` to support creating Application Configuration Service with different generation
* Add new command `az spring application-configuration-service update --generation` to support updating Application Configuration Service to different generation
* Add new command `az spring application-configuration-service git repo add --ca-cert-name` to support binding certificate to Application Configuration Service Gen2
```

1.12.2
---
* Add default `enabled_state` for `az spring config-server set` in Standard Counsumption tier.

> >>>>>>> main

Reviewer 2: ```suggestion
```
